frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx

```diff
@@ -47,7 +47,7 @@ export const DataPermissionsHelp = () => {
           value="database-view-data-level"
           data-testid="database-view-data-level"
         >
-          <Accordion.Control>{t`Database 'View data' levels`}</Accordion.Control>
```

Comment:
zero clue why these changes from the base branch are getting included into this PR.... I think this is a github glitch as git cli and gui are showing me this:
<img width="1517" alt="Screenshot 2024-04-11 at 3 48 09 PM" src="https://github.com/metabase/metabase/assets/7104357/bc6b7d4b-3f29-40a3-b053-2852478982da">